package.json

```diff
@@ -17,6 +17,7 @@
     "lint": "npm run lint:server",
     "lint:server": "DEBUG=eslint:cli-engine npx eslint --max-warnings=0 --ext .js,.jsx .",
     "lint:static-site": "cd static-site && DEBUG=eslint:cli-engine npx eslint --max-warnings=0 --ext .js,.jsx,.ts,.tsx .",
+    "type-check": "cd static-site && tsc",
```

Comment:
@genehack could you look into this [type error](https://github.com/nextstrain/nextstrain.org/actions/runs/11826364406/job/32952189773#step:5:10)? It's related to changes from https://github.com/nextstrain/nextstrain.org/pull/1032

```
Error: components/nav/nav-logo.tsx(25,12): error TS2322: Type '{ src: string; }' is not assignable to type 'string | StaticImport'.
  Type '{ src: string; }' is missing the following properties from type 'StaticImageData': height, width
```

Reply:
yep, will try to look at that tomorrow.